tests/e2e/helpers.go

```
@@ -0,0 +1,53 @@
+	"net"
+	"net/url"
+	"time"
+)
+
+// pingTCP checks if a TCP port is accepting connections.
+func pingTCP(addr string, timeout time.Duration) error {
+	conn, err := net.DialTimeout("tcp", addr, timeout)
+	if err != nil {
+		return fmt.Errorf("tcp dial %s: %w", addr, err)
+	}
+	conn.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)